types/jasmine/jasmine-tests.ts

```diff
@@ -115,6 +115,18 @@ describe("Included matchers:", () => {
         expect(foo).not.toBeFalsy();
     });
 
+    it("The 'toBeTrue' matcher is for matching with true", () => {
+        expect(true).toBeTrue();
+        expect(false).not.toBeTrue();
+        expect({}).not.toBeTrue();
+    });
+
+    it("The 'toBeTrue' matcher is for matching with true", () => {
```

Comment:
Should be `"The 'toBeFalse' matcher is for matching with false"`

Reply:
Nice catch! I've fixed it.